ingest/src/test/java/com/connexta/ingest/IngestApplicationIntegrationTest.java

```diff
@@ -49,9 +55,59 @@ public void beforeEach() {
     mvc = MockMvcBuilders.webAppContextSetup(wac).build();
   }
 
+  @After
+  public void afterEach() {
```

Comment:
Spring doesn't manage our mock lifecycles, so we need to reset them between tests so we don't cross contaminate.

Reply:
✏️ Maybe move this to the `@Before` method instead

Reply:
I think I remember there being a possible timing issue when you reset mocks before a test.